crud/create.js

```diff
@@ -0,0 +1,45 @@
+module.exports = (binding) => {
+    let columns = binding.fields.filter(f => !f.identity).map(f => `[${f.column}]`).join(', ');
+    return `
+    CREATE PROCEDURE ${binding.spName}
+        @data ${binding.tt} READONLY
+    AS
+    SET NOCOUNT ON
+    DECLARE @result ${binding.tt}
+    BEGIN TRY
+        INSERT INTO ${binding.name} (${columns})
+        OUTPUT INSERTED.* INTO @result
+        SELECT ${columns}
+        FROM @data
+
+        SELECT 'data' AS resultSetName

```

Comment:
Can we discuss if 'data' is good as name? If we call multiple procedures as part of a bigger one, it is better each to return  different name for the resultset.

Reply:
Sure. Instead of 'data' it can be the table name. e.g. 'customer.customer', 'customer.organization', etc.